orochi/asyncproc.py

```diff
@@ -256,9 +266,9 @@ def __reader(self, collector, source):
         while True:
             data = os.read(source.fileno(), 65536)
             self.__lock.acquire()
-            collector.append(data)
+            collector.append(data.decode("ascii"))
```

Comment:
you're totally right, I've put out of the locked zone, and it would have saved me a bug investigation; because `data == ""` could not be True when data was bytes, so the thread never quit and I had 100% usage when stopping the player!

I've fixed that and squashed the commits.